src/main/java/io/confluent/connect/jdbc/TimestampIncreasingTableQuerier.java

```diff
@@ -92,33 +94,51 @@ protected void createPreparedStatement(Connection db) throws SQLException {
       //  timestamp 1235, id 22
       //  timestamp 1236, id 23
       // We should capture both id = 22 (an update) and id = 23 (a new row)
-      builder.append(" WHERE (\"");
+      builder.append(" WHERE (");
+      builder.append(quoteString);
```

Comment:
Not sure which is better since the string builder also avoids lots of extra copying that we'll get with a quoteString helper, but this is rarely executed anyway so I've changed to to make it less error prone.